src/os/ui/ogc/ogcserver.js

```diff
@@ -821,6 +843,64 @@ os.ui.ogc.OGCServer.prototype.displayConsentAlerts = function(result) {
 };
 
 
+/**
+ * Extract relevant information from the GetCapabilities result for display in Descriptors, HoverText, etc
+ * @param {string} constraints
+ * @return {?Array<string>}
+ */
+os.ui.ogc.OGCServer.prototype.getProperties = function(constraints) {
+  var properties = [];
+  var xmlDocument = null;
+
+  if (constraints && constraints.indexOf('xml version') && window.DOMParser) {
```

Comment:
is there a specific reason for calling to `DOMParser` here? `os.xml.loadXml` is our typical call to parse a document from XML. the `os.xml` namespace also has a bunch of other utility functions for XML (like escaping).

Reply:
In this doc it was all `goog.dom.xml`; so I just was using the "vanilla JS" way for this new code. I'll update to use the `os.xml` in the final version.